app/controllers/budgets_controller.rb

```diff
@@ -32,6 +32,8 @@ def create
   def edit
     @budget = Budget.find(id)
     @budget_presenter = BudgetPresenter.new(@budget)
+    @current_value = @budget_presenter.value
```

Comment:
I'm not sure about this - do we not want to show the user the invalid value they tried to enter so they can correct it here?

Reply:
Ah sorry, I've just tested this and see what you mean! Yep, this is definitely an improvement over:

<img width="867" alt="image" src="https://user-images.githubusercontent.com/19826940/228861252-595cb371-0cfb-46cc-aa1b-65897d8e82cf.png">